nosso-calendario/src/main/java/com/br/devefc/nossocalendario/model/User.java

```diff
@@ -32,15 +32,20 @@
 	@NotEmpty
 	@Size(min = 6)
 	private String senha;
+	
+	@NotBlank
+	@NotEmpty
+	private String nome;
 
 	private LocalDateTime acesso_cadastro  = LocalDateTime.now();
 
 	public User() {
 	}
 
-	public User(@NotBlank @NotEmpty @Email String login,EncodeSenha senha) {
+	public User(@NotBlank @NotEmpty @Email String login,EncodeSenha senha,@NotBlank @NotEmpty String nome) {

```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

NotBlank já faz o que o NotEmpty faz :).